test/integration/users.test.js

```diff
@@ -1503,6 +1503,42 @@ describe("Users", function () {
     });
   });
 
+  describe("PATCH /archived", function () {
+    beforeEach(async function () {
+      const rolesToBeAdded = {
+        archived: false,
+        in_discord: false,
+      };
+      await addUser({ ...userData[0], roles: rolesToBeAdded });
+    });
+
+    afterEach(async function () {
+      await cleanDb();
+    });
+
+    it("returns successful response", function (done) {
+      chai
+        .request(app)
+        .patch("/users/archived")
+        .set("cookie", `${cookieName}=${superUserAuthToken}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(200);
+          expect(res.body.data).to.have.property("totalUsersArchived");
+          expect(res.body.data).to.have.property("totalOperationsFailed");
+          expect(res.body.data.totalUsersArchived).to.be.greaterThan(0);
+          expect(res.body.data.totalOperationsFailed).to.be.equal(0);
```

Comment:
if we are adding 1 user to db then why are we checking greater than 0 if only 1 user exist why not assert it to 1

Reply:
tomorrow it may change 50 users and tests would still pass though the target was only 1 person